metrics/prometheus/prometheus.go

```diff
@@ -197,6 +198,10 @@ func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMet
 		"connections_opened",
 		"Count of successful connections opened to Prebid Server.")
 
+	metrics.tmaxTimeout = newCounterWithoutLabels(cfg, reg,
+		"tmax_timeout",
+		"Count of requests rejected due to timeout exceed.")
```

Comment:
lets add tmax in description. How about `Count of requests rejected due to Tmax timeout exceed.`

Reply:
updated description in [62c4b50](https://github.com/prebid/prebid-server/pull/2892/commits/62c4b507afbb8af6aabdb3e20e6cb3ec245f4761)